app/main.py

```diff
@@ -1,8 +1,21 @@
 class Person:
-    # write your code here
-    pass
+
+    people = {}
+
+    def __init__(self, name: "str", age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people.update({self.name: self})
+
+    def add_partner(self, pertner_name: str) -> None:
```

Comment:
There is a typo in the method name `add_partner`. It should be `partner_name` instead of `pertner_name`.